src/database/migrations/0059-rename-db_operation_audit_logs.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE db_operation_audit_logs RENAME TO service_query_audit_logs;
```

Comment:
While looking at my local database in another context, this is enough to rename the table, but the indices are still not renamed.